src/gui/TabManager.cc

```diff
@@ -91,15 +90,15 @@ void TabManager::closeTabRequested(int x)
   assert(tabWidget != nullptr);
   if (!maybeSave(x)) return;
 
-  auto *temp = (EditorInterface *)tabWidget->widget(x);
-  editorList.remove(temp);
-  tabWidget->removeTab(x);
+  auto *closingEditor = (EditorInterface *)tabWidget->widget(x);
```

Comment:
I belive we should try to use `qobject_cast<EditorInterface*>(tabWidget->widget(x))`, perhaps with a nullptr assert here.

Reply:
OK.